### PR TITLE
update the roadmap description

### PR DESCRIPTION
**Pull request by Rozzii**
This commit updates the roadmap documentation and expands it according to current practices.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,56 +1,75 @@
...
+Baremetal Operator, Cluster API Provider Metal3, IP Address Manager,
 Ironic image and other Metal3 components. In addition there is
-a `Feature Requests` column that contains items that have not yet been
+a `No status` column that contains items that have not yet been
 accepted and the `Backlog` column that contains items that have been accepted
-but not yet planned for a specific release.
+but work has not started on them.
 
 An issue can be planned for a specific release if someone volunteers to take
 ownership of the feature. The owner will then be assigned the issue. An owner
```
Comment (tuminoid) [inline ROADMAP.md:16]:
```suggestion
ownership of the issue. The owner will then be assigned the issue. An owner
```
Not all issues on roadmap are features, if we put all new issues there as you suggested yesterday.

File: ROADMAP.md [hunk 2]
```diff
@@ -1,56 +1,75 @@
...
 does not have to carry the whole design and implementation processes on her own
 but must instead make sure that the feature is being worked on and will be
 completed by the planned release date.
```
Comment (tuminoid) [inline ROADMAP.md:19]:
```suggestion
completed by the planned milestone.
```
Maybe milestone is better as we take them into use now?

File: ROADMAP.md [hunk 3]
```diff
@@ -1,56 +1,75 @@
...
-The Roadmap of the Metal3 project is discussed mainly in community meetups.
-Meetups are usually long discussions within the community members and the
-specific focus is to discuss long term goals and Roadmap for the project.
+## Ownership of the Roadmap
 
-First Metal3 meetup was arranged on 15th of May 2023. Here is the
-[link](https://docs.google.com/document/d/1Exm5k6JB2JlD9_FOaM68R2ASHi7ZsryF_xEqLG1j4zA/edit)
-of the meeting minutes and discussion topics.
+The Roadmap update and maintenance is performed by the maintainers of the
+Metal3 project. And the "Roadmap editors" team.
```
Comment (tuminoid) [inline ROADMAP.md:75]:
```suggestion
The Roadmap update and maintenance is performed by the "Roadmap editors" of the
Metal3 project.
```
Maintainers are part of the editors implicitly.

File: ROADMAP.md [hunk 4]
```diff
@@ -1,56 +1,75 @@
...
+The priorities of the project are iteratively refined and adjusted during
+weekly meetings that are open to all community members. During the weekly
+community meetings contributors can propose and discuss new features,
+bugs, adjustments to release timelines, thus the roadmap of the project is
+under constant maintenance.
+
 ## Proposing a feature
 
-Proposing a new feature for a specific release of one of the components is done
+Proposing a new feature for a of one of the components is done
```
Comment (mboukhalfa) [inline ROADMAP.md:43]:
```suggestion
Proposing a new feature for one of the components is done
```

File: ROADMAP.md [hunk 5]
```diff
@@ -1,56 +1,75 @@
...
 by opening an issue in the [metal3-docs](https://github.com/metal3-io/metal3-docs)
 repository, describing the feature and which components and release are
-targeted. The new issue will automatically appear in the feature request
-column of the roadmap.
+targeted. The new issue will automatically appear in the "No status"
+column of the roadmap and will stay there until it is  are triaged and accepted.
```
Comment (mboukhalfa) [inline ROADMAP.md:47]:
```suggestion
column of the roadmap and will stay there until it is triaged and accepted.
```

File: ROADMAP.md [hunk 6]
```diff
@@ -1,56 +1,75 @@
...
 
-The Roadmap of the Metal3 project is discussed mainly in community meetups.
-Meetups are usually long discussions within the community members and the
-specific focus is to discuss long term goals and Roadmap for the project.
+## Ownership of the Roadmap
 
-First Metal3 meetup was arranged on 15th of May 2023. Here is the
-[link](https://docs.google.com/document/d/1Exm5k6JB2JlD9_FOaM68R2ASHi7ZsryF_xEqLG1j4zA/edit)
-of the meeting minutes and discussion topics.
+The Roadmap update and maintenance is performed by "Roadmap editors" of the
```
Comment (tuminoid) [inline ROADMAP.md:74]:
```suggestion
The Roadmap update and maintenance is performed by "Roadmap editors" team of the
```
Sorry, missed a word in my previous suggestion

**Review by kashifest — COMMENTED** [review]
/approve
Great work @Rozzii

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**